element-connector/src/main/java/org/eclipse/californium/elements/auth/AbstractExtensiblePrincipal.java

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Please replace that with the californium headers.
We may change that according EPL-2.0 after the 2.0 release.

Reply:
sorry, settings from my Hono workspace keep getting in the way ...